src/EA.Weee.RequestHandlers/Mappings/EvidenceNoteMap.cs

```diff
@@ -38,7 +38,9 @@ public EvidenceNoteData Map(Note source)
                 Protocol = source.Protocol.HasValue ? (Protocol?)source.Protocol.Value : null,
                 WasteType = source.WasteType.HasValue ? (WasteType?)source.WasteType.Value : null,
                 EvidenceTonnageData = source.NoteTonnage.Select(t =>
-                    new EvidenceTonnageData(t.Id, (WeeeCategory)t.CategoryId, t.Received, t.Reused)).ToList(),
+                    new EvidenceTonnageData(t.Id, (WeeeCategory)t.CategoryId, t.Received, t.Reused,
+                    t.NoteTransferTonnage != null ? t.NoteTransferTonnage.Where(ntt => ntt.TransferNote.Status.Value != Domain.Evidence.NoteStatus.Rejected.Value && ntt.TransferNote.Status.Value != Domain.Evidence.NoteStatus.Void.Value).Select(ntt => ntt.Received).Sum() : null,
```

Comment:
tempted to put the Rejected and voided status into a local list and then do something like !excludeStatus.Contains(transferNote.Status.Value)